webpack.config.js

```diff
@@ -1,17 +1,11 @@
 
 const NODE_ENV = process.env.NODE_ENV;
```

Comment:
we can probably have a webpack.config.base to include the duplicate codes in both dev and production config files.